File: deno/src/dept/dept.resolver.ts
import {
  _internals as deptService,
} from "./dept.service.ts";

export const _internals = {
  deptLoginSelect,
};

async function deptLoginSelect(
  dept_id: string,
) {
  const token = await deptService.deptLoginSelect(dept_id);
  return token;
}

File: rust/src/src/base/dept/dept_resolver.rs
use anyhow::Result;
use crate::common::context::Ctx;

use super::dept_service;

pub async fn dept_login_select<'a>(
  ctx: &mut impl Ctx<'a>,
  dept_id: String,
) -> Result<String> {
  let res = dept_service::dept_login_select(ctx, dept_id).await?;
  Ok(res)
}
